src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,102 @@
 import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tab,
+  Tabs,
+} from 'carbon-components-react';
+
+const props = {
+  tabs: {
+    selected: 0,
+    triggerHref: '#',
+    role: 'navigation',
+  },
+  tab: {
+    href: '#',
+    role: 'presentation',
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--col-lg-16">
+          <Breadcrumb noTrailingSlash aria-label="Page navigation">
+            <BreadcrumbItem>
+              <a href="/">Getting Started</a>
+            </BreadcrumbItem>
+          </Breadcrumb>
+          <h1 className="landing-page__heading">
+            Design &amp; build with Carbon
+          </h1>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r2">
+        <div className="bx--col bx--no-gutter">
+          <Tabs {...props.tabs} aria-label="Tab navigation">
+            <Tab {...props.tab} label="About">
+              <div className="bx--grid bx--grid--no-gutter bx--grid--full-width">
+                <div className="bx--row landing-page__tab-content">
+                  <div className="bx--col-md-4 bx--col-lg-7">
+                    <h2 className="landing-page__subheading">
+                      What is Carbon?
+                    </h2>
+                    <p className="landing-page__p">
+                      Carbon is IBM’s open-source design system for digital
+                      products and experiences. With the IBM Design Language as
+                      its foundation, the system consists of working code,
+                      design tools and resources, human interface guidelines,
+                      and a vibrant community of contributors.
+                    </p>
+                    <Button>Learn more</Button>
+                  </div>
+                  <div className="bx--col-md-4 bx--offset-lg-1 bx--col-lg-8">
+                    <img
+                      className="landing-page__illo"
+                      src="{`${process.env.PUBLIC_URL}/tab-illo.png`}"
```

Comment:
Your image is not displaying - it should not be contained in double quotes: 
`src={`${process.env.PUBLIC_URL}/tab-illo.png`}`